contracts/rollup.sol

```diff
@@ -107,17 +106,15 @@ contract RollupHelpers is RollupSetup {
         bytes32 _updatedRoot,
         Types.Usage batchType
     ) internal {
-        Types.Batch memory newBatch = Types.Batch({
-            stateRoot: _updatedRoot,
-            accountRoot: accountsTree.getTreeRoot(),
-            depositTree: ZERO_BYTES32,
-            committer: msg.sender,
-            txRoot: txRoot,
-            stakeCommitted: msg.value,
-            finalisesOn: block.number + governance.TIME_TO_FINALISE(),
-            timestamp: now,
-            batchType: batchType
-        });
+        Types.Batch memory newBatch;
+        newBatch.stateRoot = _updatedRoot;
+        newBatch.accountRoot = accountsTree.getTreeRoot();
+        // newBatch.depositTree default initialized to 0 bytes
+        newBatch.committer = msg.sender;
+        newBatch.txRoot = txRoot;
+        newBatch.finalisesOn = block.number + governance.TIME_TO_FINALISE();
+        // newBatch.withdrawn default initialized to false
+        newBatch.batchType = batchType;
```

Comment:
I compared these two different styles of assignment and the incoming one is slightly more gas efficient.


```solidity
pragma solidity ^0.5.15;


contract Foo {
    struct A {
        bytes32 a;
        bytes32 b;
        bytes32 c;
    }
    // 478 gas
    function createStruct () public pure{
        A memory a;
        a.a = keccak256("123");
        a.b = keccak256("456");
  
    }
    // 535 gas
    function createStruct2() public pure {
        A memory a = A({
            a: keccak256("123"),
            b: keccak256("456"),
            c: bytes32(0x00)
        });
    }
}
```

Reply:
Perfect!